.github/workflows/coffee.yml

```
@@ -19,22 +19,18 @@ jobs:
       - name: Setup Go environment
         uses: actions/setup-go@v5
 
-      - name: Run Test and Send Slack Message
-        run: go run main.go coffee-break
-        env:
-          SLACK_TOKEN: ${{ secrets.SLACK_TOKEN }}
-          HACBS_CHANNEL_ID: ${{ secrets.HACBS_CHANNEL_ID }}
-
-      - name: Commit and push
+      - name: Run Script, Commit and push
         run: |
-          git config --global --add safe.directory /__w/qe-tools/qe-tools
-          git init /__w/qe-tools/qe-tools
           git config user.name "GitHub Action"
           git config user.email "action@github.com"
           git switch -c coffee-break
+          go run main.go coffee-break
           git add config/coffee-break/last_week.txt
           git commit -m "chore: Update config/coffee-break/last_week.txt"
```

```suggestion
          git commit -m "chore: update config/coffee-break/last_week.txt"
```
This should make commitlint stop complaining 
```
chore: Update config/coffee-break/last_week.txt
Co-authored-by: GitHub Action <action@github.com>
✖   subject must not be sentence-case, start-case, pascal-case, upper-case [subject-case]
```
https://github.com/redhat-appstudio/qe-tools/actions/runs/8264551331/job/22608387601

---

Will update